App/Files/FilesUtility/readAndWriteToFiles.js

```diff
@@ -0,0 +1,11 @@
+function* writeToTextFile() {
+    fs.writeFile('Output.txt', data, (err) => {
+
+        // In case of a error throw err. 
+        if (err) throw err;
+    })
+}
+
+function readFrom() {
+
+}
```

Comment:
make sure to export both these functions to be used in other files

`module.exports = {writeToTextFile, readFromTextFile}`